.github/ISSUE_TEMPLATE/membership.md

```diff
@@ -14,14 +14,40 @@ assignees: ''
 
 e.g. (at)example_user
 
-### Requirements
+### Membership level
+
+Membership level according to https://github.com/keptn/community/blob/master/COMMUNITY_MEMBERSHIP.md
+
+member | approver | maintainer
+
+### Requirements (for member)
 
 - [ ] I have reviewed the community membership guidelines (https://github.com/keptn/community/blob/master/COMMUNITY_MEMBERSHIP.md)
 - [ ] I have enabled 2FA on my GitHub account. See https://github.com/settings/security
 - [ ] I have subscribed to the [Keptn Slack channel](http://slack.keptn.sh/)
 - [ ] I am actively contributing to 1 or more Keptn subprojects
 - [ ] I have two sponsors that meet the sponsor requirements listed in the community membership guidelines. Among other requirements, sponsors must be approvers or maintainers of at least one repository in the organization and not both affiliated with the same company
-- [ ] I have spoken to my sponsors ahead of this application, and they have agreed to sponsor my application
+- [ ] I have spoken to 2 sponsors (approvers or maintainers) ahead of this application, and they have agreed to sponsor my application
+
+### Requirements (for approver, if applicable)
+
+- [ ] I am a reviewer of the codebase for at least 1 month
+- [ ] I am a reviewer for/or author of substantial PRs to the codebase, with the definition of substantial subject to the maintainer's discretion (e.g., refactors/adds new functionality rather than one-line changes).
+- [ ] I have spoken to 2 sponsors (maintainers) ahead of this application, and they have agreed to sponsor my application
+
+### Requirements (for maintainer, if applicable)
+
+- [ ] I have deep understanding of the technical goals and direction of the subproject
+- [ ] I have deep understanding of the technical domain (specifically the language) of the subproject
+- [ ] I did sustained contributions to design and direction by doing all of:

```

Comment:
Can we change it to: (to have sub-items)
```
- [ ] I did sustained contributions to design and direction by doing all of:
  - [ ] Authoring and reviewing proposals
  - [ ] Initiating, contributing, and resolving discussions (e-mails, GitHub issues, meetings)
  - [ ] Identifying subtle or complex issues in designs and implementation PRs
```

... like it is done in the `COMMUNITY_MEMBERSHIP.md`

Reply:
done